### PR TITLE
Data loading is too slow #217

### PR DESCRIPTION
**Pull request by eladkarakuli**
Using SQLAlchemy Core for better INSERT performance. Tested with complete accidents file, while getting an error :
> finished processing all directories, except: static/data/lms/Accidents Type 1/2007104H1: x and y coordinates are missing from the accidents file!
>  static/data/lms/Accidents Type 1/2005104H1: x and y coordinates are missing from the accidents file!"

Results are - 
Total rows imported: 619815
Total seconds: 255

### CODE REVIEW

**Comment by danielhers** [comment]
Thanks, 255 seconds is definitely faster than before!
The error is known. Issue #136 is about it.
Do you think you could [squash your commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html) so as not to have several changes done and then undone in subsequent commits?

File: process.py [hunk 1]
```diff
@@ -210,27 +210,17 @@ def import_to_datastore(directory, provider_code, batch_size):
...
-                print("\rcommitting ({0} items done)...".format(i))
-                db.session.commit()
-                print("commited.")
-
-        # commit any left sessions, if any were imported
-        if imported > 0:
-            db.session.commit()
-
+        acc = list(import_accidents(provider_code=provider_code, **files_from_lms))
+        print(int((datetime.now() - now).total_seconds()))
```
Comment (danielhers) [inline process.py]: Please make this line more informative - mention that's it's how long (in seconds) it took...

File: process.py [hunk 2]
```diff
@@ -150,19 +150,19 @@ def import_accidents(provider_code, accidents, streets, roads):
...
-            address=get_address(accident, streets),
-            latitude=lat,
-            longitude=lng,
-            type=Marker.MARKER_TYPE_ACCIDENT,
-            subtype=int(accident[field_names.accident_type]),
-            severity=int(accident[field_names.accident_severity]),
-            created=parse_date(accident),
-            locationAccuracy=int(accident[field_names.igun]),
-        )
+        marker = {
```
Comment (danielhers) [inline process.py]: Why is it better to use a dictionary here rather than an object?

Comment (eladkarakuli) [inline process.py]: Yes, it's for SQLAlchemy Core. The *db.session.*execute method, takes the values for the insert command as a dictionary.

File: process.py (hunk 1 above)
Comment (eladkarakuli) [inline process.py]: I wanted to separate timing between the accidents import, and the db commit execution, for testing.
line removed, thanks for the comment.

**Comment by eladkarakuli** [comment]
Hey @danielhers, I created a [new pull request](https://github.com/hasadna/anyway/pull/237) for the fix.
You can close this one.

**Pull request closed without merging**